### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**
Dependency updates

### CODE REVIEW

File: .github/workflows/ci-build-next-java.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Run tests and build with Maven
+        run: mvn --batch-mode --update-snapshots clean package -DtrimStackTrace=false
+      - name: Publish Test Report
+        uses: scacap/action-surefire-report@v1
+        if: ${{ always() }}
```
Comment (morazow) [inline .github/workflows/ci-build-next-java.yml:30]: I think this is not the latest version of project-keeper, in the last one there are additional conditions

File: doc/changes/changes_0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Virtual Schema for document data on BucketFS 0.5.0, released 2021-11-23
```
Comment (morazow) [inline doc/changes/changes_0.5.0.md:1]:
```suggestion
# Virtual Schema for document data on BucketFS 0.5.0, released 2021-11-26
```

File: pom.xml [hunk 3]
```diff
@@ -9,17 +9,17 @@
...
         <!--
-        M4 does not fail tests if beforeAll fails
-        M5 fails in integration tests with a strange error
-        -->
+            M4 does not fail tests if beforeAll fails
+            M5 fails in integration tests with a strange error
+            -->
         <surefire.and.failsafe.plugin.version>3.0.0-M3</surefire.and.failsafe.plugin.version>
-        <vs-commom-document-files.version>2.2.0</vs-commom-document-files.version>
+        <vs-commom-document-files.version>4.0.0-SNAPSHOT</vs-commom-document-files.version>
```
Comment (morazow) [inline pom.xml:16]: Do not forget to update version once you release vs-common-document-files.

**Review by morazow — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (jakobbraun) [reply pom.xml:16]: :heavy_check_mark:

**Review by morazow — APPROVED** [review]

**Pull request merged**